### DISCUSSION TITLE
Cloning Gist with human name

### DISCUSSION THREAD

**Discussion by marcelomgarcia** [community/community#62201] [Other Feature Feedback, Questions, & Ideas]
Dear all,

I think should be clear in the documentation how to clone a gist with a human name instead of a hash:

```
PS C:\Users\marcelo\Work> git clone https://gist.github.com/b9186eed2601dea0c16b8a0dc2e8b4ec.git learning_azure
```

This creates a directory with name `learning_azure` instead of `b9186eed...` Why is this not clear in the [documentation](https://docs.github.com/en/get-started/writing-on-github/editing-and-sharing-content-with-gists/forking-and-cloning-gists)?

**Comment by tuves** [comment]
Welcome to the GitHub Community, @marcelomgarcia, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Reply by marcelomgarcia** [reply]
Hi, indeed I created this ticket in the wrong place. Thank you for your note.
